src/com/fairphone/fplauncher3/Launcher.java

```diff
@@ -1123,9 +1129,9 @@ protected void onPause() {
     }
 
     protected static boolean hasSettings() {
-        return false;
+        return true;
     }
-    
+
```

Comment:
Whitespace, whitespace. Leave it, but next time make sure it doesn't get in the way, please, :)

Reply:
Sorry, Atom does this automatically ^^

But if it bothers you when reviewing the PR, you can see the diff without the whitespace changes here: https://github.com/WeAreFairphone/FP2-Launcher/pull/27/files?w=1

Reply:
Oh! Good find, thanks!
I use Atom too. I think it's a good idea to remove whitespace, but in a dedicated commit, as a good Git practice. The whitespace changes get in the way with complex merges and rebases (although we're not doing any of that here). The Git package for Atom is great because it lets you easily select lines to commit or suppress (instead of the painful `git add -p` CLI).